main.cpp

```diff
@@ -1,11 +1,14 @@
 #include <QApplication>
+#include <QDebug>
```

Comment:
Why include QDebug here? Let's postpone that, if needed, to the last minute when it's required.